templates/form_themes/app_newsletter_subscription_theme.html.twig

```diff
@@ -0,0 +1,26 @@
+{%- block app_newsletter_subscription_widget -%}
```

Comment:
IMO It's not really one form theme, but juste one template which can be included.

Reply:
I agree. The only time I needed to do such a trick it was because I needed a collection a form rendered correctly.
But in that case, the form_theme should be applied to a single file and not globally IMO.

Could you explain why you did that **exactly** ?

Reply:
I do that because it's the simplest way to custom a form theme when this form is used multiple times.

Reply:
what about [include](https://twig.symfony.com/doc/2.x/tags/include.html) your template of form without declare it as form theme?

Reply:
Yes please, form themes are not designed for that. They should not contain any content or anything like this. It's purely HTML structure. For example if you want to render the same field with another label you will need a new local form theme, what a pain.